Framework/Kernel/test/TimeSeriesPropertyTest.h

```diff
@@ -382,20 +389,30 @@ class TimeSeriesPropertyTest : public CxxTest::TestSuite {
     TimeROI *rois = new TimeROI;
     rois->addROI("2007-11-30T16:17:00", "2007-11-30T16:17:31");
     this->assert_two_vectors(log->filteredValuesAsVector(rois), log->valuesAsVector(), 0.01);
+    this->assert_two_vectors(log->filteredTimesAsVector(rois), log->timesAsVector());
```

Comment:
A general comment on this method.  There is no need to allocate the `TimeROI` object on the heap. The vector elements of the `m_roi` class member of `TimeROI` will be allocated on the heap anyway.  So it's enough to declare it on the stack:
`TimeROI rois;`
But if we do allocate it on the heap, then we need to delete it at the end.